source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj

```diff
@@ -21,10 +21,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Common" Version="0.33.0" PrivateAssets="All" />
-    <PackageReference Include="Cake.Core" Version="0.33.0" PrivateAssets="All" />
-    <PackageReference Include="NuGet.Packaging" Version="5.0.2" PrivateAssets="All" />
-    <PackageReference Include="NuGet.Protocol" Version="5.0.2" PrivateAssets="All" />
+    <PackageReference Include="Cake.Common" Version="0.38.5" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="0.38.5" PrivateAssets="All" />
+    <PackageReference Include="NuGet.Packaging" Version="5.4.0" PrivateAssets="All" />
+    <PackageReference Include="NuGet.Protocol" Version="5.4.0" PrivateAssets="All" />
```

Comment:
LGTM and [matches the NuGet version loaded by Cake v0.38.5](https://github.com/cake-build/cake/blob/v0.38.5/src/Cake.NuGet/Cake.NuGet.csproj#L22-L27). Once Cake v1.0 goes live, we should update this addin again to target the version that Cake v1.0 ships with.

Reply:
I'll gladly update again, no problem